packages/core-contracts/contracts/utils/ERC20.sol

```diff
@@ -0,0 +1,93 @@
+//SPDX-License-Identifier: UNLICENSED
+
```

Comment:
nit: Not using spaces here in other contracts

Reply:
fixed